### ISSUE TITLE
unable to build docker image

### ISSUE THREAD

**Issue by xolom** [bnb-chain/bsc#1270]
There is an issue with an alpine package when building the docker image

```
#0 0.238 fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/x86_64/APKINDEX.tar.gz
#0 0.513 fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/x86_64/APKINDEX.tar.gz
#0 0.696 ERROR: unable to select packages:
#0 0.720   bind-tools-9.16.36-r0:
#0 0.720     breaks: world[bind-tools~9.16.33]
```

Seems line the package version needs to be updated (https://github.com/bnb-chain/bsc/blob/master/Dockerfile#L29) ?

**Comment by xolom** [comment]
I could fix it with https://github.com/bnb-chain/bsc/pull/1271

**Comment by KeefeL** [comment]
> I could fix it with #1271

There is a Lint Commit Messages error, could you please fix it?
